pkg/kn/commands/config/config.go

```diff
@@ -0,0 +1,81 @@
+// Copyright © 2022 The Knative Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package configCmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"gopkg.in/yaml.v2"
+	"knative.dev/client/pkg/kn/commands"
+)
+
+var cfgCmdExamples = ``
+
+var ConfigKeyNotFoundErrorMsg = "Could not find %q in the configuration file"
+
+func NewConfigCommand(p *commands.KnParams) *cobra.Command {
+	cfgCmd := &cobra.Command{
+		Use:     "config COMMAND",
+		Short:   "Manage Knative configuration file",
+		Aliases: []string{"cfg"},
+		Example: cfgCmdExamples,
+	}
+	cfgCmd.AddCommand(NewConfigGetCommand(p))
+	cfgCmd.AddCommand(NewConfigSetCommand(p))
+	cfgCmd.AddCommand(NewConfigShowCommand(p))
+	cfgCmd.AddCommand(NewConfigSinkMappingCommand(p))
+	cfgCmd.AddCommand(NewConfigChannelTypeMappingCommand(p))
+	return cfgCmd
+}
+
+func defaultPrinter(cmd *cobra.Command, list []interface{}) error {
+	b, err := yaml.Marshal(list)
+	if err != nil {
+		return err
+	}
+	if _, err := fmt.Fprint(cmd.OutOrStdout(), string(b)); err != nil {
+		return err
+	}
+	return nil
+}
+
+func viperUpdater(viper *viper.Viper, cfgKey string) func([]interface{}) error {
```

Comment:
Can we have some comments to these... I know they are private but brief one liner on what function does helps reader who was not aware of that function. Tx